doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,133 @@
+.. _ug_nrf70_developing_fw_patch_ext_flash:
+
+Firmware patches in the external flash
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Overview
+********
+
+The nRF70 Series firmware patch is memory intensive and is by default part of the internal flash.
+In order to save flash memory for resource constrained hosts, the nRF70 Series offers a feature to move firmware patches to the external flash, if the external flash is available.
```

Comment:
Suggestion for a more generic wording:

"The firmware patch includes code that is executed on the nRF70 Series device. The size of the firmware patch may be considerably large and limits the amount of on-chip code memory available for the user application; in order to increase the amount of on-chip non-volatile memory available for user applications, the nRF70 Series Wi-Fi driver supports the option of relocating the device FW patch to external non-volatile memory, in case that is available.